src/fides/api/alembic/migrations/versions/d0cbfec0b2dd_add_okta_to_connection_types.py

```diff
@@ -0,0 +1,61 @@
+"""add okta to connection_types
+
+Revision ID: d0cbfec0b2dd
+Revises: c9c72b3d550b
+Create Date: 2025-04-23 15:22:57.920715
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "d0cbfec0b2dd"
+down_revision = "c9c72b3d550b"
+branch_labels = None
+depends_on = None
+
+type_to_handle = "okta"
+
+
+def get_enum_values():
+    conn = op.get_bind()
+    result = conn.execute("SELECT unnest(enum_range(NULL::connectiontype))").fetchall()
+    return [row[0] for row in result]
```

Comment:
I think we should do the `row[0].strip()` here instead of in the big list comprehension

```suggestion
    return [row[0].strip() for row in result]
```